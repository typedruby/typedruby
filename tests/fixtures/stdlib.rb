# @typedruby
# Ensure that all our stdlib stubs actually load
require "bigdecimal"
require "cgi"
require "date"
require "delegate"
require "digest"
require "digest/sha1"
require "enumerator"
require "erb"
require "etc"
require "fileutils"
require "json"
require "net/http"
require "openssl"
require "optparse"
require "ostruct"
require "pathname"
require "psych"
require "rbconfig"
require "resolv"
require "rubygems"
require "securerandom"
require "set"
require "shellwords"
require "socket"
require "stringio"
require "tempfile"
require "thread"
require "tmpdir"
require "uri"
require "zlib"
