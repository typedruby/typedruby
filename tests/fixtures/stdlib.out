info: Typechecking...
success: Typecheck passed
